packages/react-ui/src/index.js

```diff
@@ -15,6 +15,7 @@ export { Alert } from "./Alert";
 export { Button } from "./Button";
 export { CardList } from "./CardList";
 export { Grid, GridCell } from "./Grid";
+export { NakedList } from "./NakedList";
```

Comment:
NakedList :/ why not List or FlatList ?

Reply:
Je sais pas moi ! J'aime bien imaginer une liste à poil ! Non `List` ça me semblait trop générique. `Flat` pourquoi pas oui. `Unstyled` ça va pas parce que si on la style ça donnerait `StyledUnstyledList` 😄

Reply:
Je pars sur Flat pour le naming